crmsh/log.py

```diff
@@ -311,27 +311,19 @@ def line_number(self):
         finally:
             self.reset_lineno(self.__save_lineno)
 
-    def status_done(self):
-        """
-        Log info message "done" in console
-        """
-        with self.raw_msg():
```

Comment:
The `raw_msg` function, and the `raw_msg` parameter in `ConsoleCustomFormatter` class, should also be removed

Reply:
And please remove `status_progress` function in bootstrap.py, it also make mess like:
```
INFO: BEGIN Waiting for cluster ...
...........INFO: END Waiting for cluster.
```

Reply:
> And please remove `status_progress` function in bootstrap.py, it also make mess like:

Instead of removing `status_progress`, we can make a progress bar which occupies a whole line of a terminal, so that logs will not interleave with it.

Reply:
OK